client/wildcard/src/components/Badge/ProductStatusBadge.story.tsx

```diff
@@ -3,6 +3,8 @@ import { Meta } from '@storybook/react'
 import { BrandedStory } from '@sourcegraph/branded/src/components/BrandedStory'
 import webStyles from '@sourcegraph/web/src/SourcegraphWebApp.scss'
 
+import { Typography } from '..'
```

Comment:
Let's use direct imports to avoid cyclic import graphs inside the' wildcard' package. And let's move the `Typography` export to `client/wildcard/src/components/Typography/index.ts` from `client/wildcard/src/components/index.ts`.

This comment is applicable to other `Typography` inside the `wildcard` package.

```suggestion
import { Typography } from '../Typography'
```